src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,147 @@
+import { ChangeEvent, useContext } from 'react';
+import styled from 'styled-components';
+import { formFieldContext } from '../../Me/components/FormField/formContext';
+
+type SwitchProps = {
+  id?: string;
+  label: string;
+  isChecked: boolean;
+  size?: 'small' | 'normal';
+  onToggle(isChecked: boolean): void;
+};
+
+const SwitchContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+let SwitchInputRoot = styled.div<Pick<SwitchProps, 'size'>>`
+  height: 20px;
+
+  input[type='checkbox'] {
+    height: 0;
+    width: 0;
+    visibility: hidden;
+
+    &:checked + label {
+      background-color: var(--tag-color);
+
+      &:after {
+        transform: translateX(-100%);
+
+        ${props =>
+          props.size === 'small'
+            ? `
+          left: calc(100% - 2.5px);
+        `
+            : `
+          left: calc(100% - 5px);
+        `}
+      }
+    }
+  }
+
+  label {
+    display: block;
+    cursor: pointer;
+    position: relative;
+    text-indent: -9999px;
+    border-radius: 100px;
+    background-color: grey;
+    transition: background-color 0.35s ease-in-out;
+
+    ${props =>
+      props.size === 'small'
+        ? `
```

Comment:
I hate this formatting but seems like prettier and vscode recognize it as plain strings :(